configs/operations/github-events-to-s3.yml

```diff
@@ -0,0 +1,9 @@
+---
+name: Github Events To S3
+
+events:
+  - all
```

Comment:
Is all events too much at this point?

Reply:
We are looking to create a data lake so the more the better, check out the discussion following from here: https://github.com/opensearch-project/opensearch-metrics/issues/76#issuecomment-2397254973